### ISSUE TITLE
TypeError: Cannot read property 'length' of undefined

### ISSUE THREAD

**Issue by Nicolabo** [astefanutti/decktape#125]
Hi,
I found a weird issue. I got this error:

```
Remark JS plugin activated
Printing slide #6       (  6/153) ...
TypeError: Cannot read property 'length' of undefined
```

However, after I excluded from my local css file

```
@import url('https://fonts.googleapis.com/css?family=Open+Sans:400,600,700&subset=latin-ext');
```

It works. But as you could imagine, the font in my slidedeck is quite essential. Do you encounter similar error before?

Thanks,

**Comment by astefanutti** [comment]
I've never seen that error though that must come from:
https://github.com/astefanutti/decktape/blob/9a26febbf2a8e203e338ea61f83d6370f9bf6cab/plugins/remark.js#L20-L28
Would you be able to change:
https://github.com/astefanutti/decktape/blob/9a26febbf2a8e203e338ea61f83d6370f9bf6cab/decktape.js#L219
with:

    console.log(chalk`{red \n${error.stack}}`);

That should give us the exact code line (I'll add a verbose option ASAP btw).

Otherwise, if you could provide a reproducer, so that I can identify the root cause.

**Comment by Nicolabo** [comment]
```
TypeError: Cannot read property 'length' of undefined
    at font.data.glyf.forEach (/Users/miko/node_modules/decktape/decktape.js:408:43)
    at Array.forEach (<anonymous>)
    at parseFont (/Users/miko/node_modules/decktape/decktape.js:406:26)
    at Array.forEach (<anonymous>)
    at parseResources (/Users/miko/node_modules/decktape/decktape.js:444:50)
    at parseXObject (/Users/miko/node_modules/decktape/decktape.js:382:7)
    at Array.forEach (<anonymous>)
    at parseResources (/Users/miko/node_modules/decktape/decktape.js:441:53)
    at printSlide (/Users/miko/node_modules/decktape/decktape.js:449:3)
    at exportSlide (/Users/miko/node_modules/decktape/decktape.js:332:3)
```

**Comment by astefanutti** [comment]
I cannot reproduce on Mac, but could you try with changing:

https://github.com/astefanutti/decktape/blob/9a26febbf2a8e203e338ea61f83d6370f9bf6cab/decktape.js#L408

into:

    if (!f.data.glyf[i].contours || f.data.glyf[i].contours.length === 0) {

**Comment by Nicolabo** [comment]
It works! Could you tell me what happens?

**Comment by astefanutti** [comment]
During fonts consolidation, glyphs from multiple physical fonts belonging to the same logical font get merged into a consolidated font. A glyph gets merged only when it's not already present in the output consolidated font. For some reasons, it seems that in your case, the glyph contours is null instead of being empty. I've tried reproducing the error by using the same font without success. So maybe it depends on the OS, ...

**Comment by astefanutti** [comment]
I've merged the fix in version `2.6.1`.
